README.md

```diff
@@ -1,44 +1,86 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Installation
 
-## Available Scripts
+Steps to run your project locally
 
-In the project directory, you can run:
+### 1. OAuth stuff
 
-### `yarn start`
+1. [Register an OAuth app in github](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/)
+2. [Deploy an instance of Gatekeeper to heroku](https://github.com/prose/gatekeeper#deploy-on-heroku) 
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
 
-### `yarn test`
+### 2. Local preparation
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+1. `git clone https://github.com/MrRamka/github-web-client.git` 
+   
+   or `git clone git@github.com:MrRamka/github-web-client.git`
+   
+2. `cd github-web-client.git`
+3. Create `.env` file and add
 
-### `yarn build`
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+````
+REACT_APP_AUTH_API_URI=heroku-gatekeeper-url
+REACT_APP_REDIRECT_URI=http://localhost:3000/login/oauth/access_token/
+REACT_APP_CLIENT_ID=github-oauth-client-id
+````
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+4. Install all packages using `yarn` command  
 
-### `yarn eject`
+### 3. Run project
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+`yarn start`
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
Application should be open automatically, if doesn't, open [http://localhost:3000](http://localhost:3000) to view it in the browser.